src/main/drivers/timer.h

```diff
@@ -119,14 +124,22 @@ typedef struct timerHardware_s {
 #if defined(STM32F4) || defined(STM32F7)
     DMA_Stream_TypeDef *dmaRef;
     uint32_t dmaChannel;
+#elif defined(STM32H7)
+    DMA_Stream_TypeDef *dmaRef;
+    uint8_t dmaRequest;
+    uint8_t dmaIrqHandler; // XXX Should be gone (can be substituted by dmaGetIdentifier)
```

Comment:
What do we need to do to get this gone?

Reply:
The DMA support for H7 at this point is actually preliminary as it only supports DMA1 and DMA2; there are two other DMA controllers, BDMA and MDMA. While MDMA is a bit different, BDMA is essential for efficient DMA between SRAM3 and several peripherals on AHB4/APB4. To do it, we probably have to extend the DMA facility itself. This is in my todo list and partially worked on, but ETA is not yet unknown.